From: Larsen <larsen007@web.de>
Date: Wed, 25 Mar 2015 16:31:01 +0100
Subject: Checkinstall fails to add files to dpkg conffiles

checkinstall 1.6.2-4 does not tag conffiles of packages it creates as such.
Therefore, files of a created package in /etc will be deleted when the  
package is removed.

---
--- a/checkinstall
+++ b/checkinstall
@@ -2609,7 +2609,7 @@
 fi
 
 # Tag files in /etc to be conffiles
-find -L $BUILD_DIR/etc -type f 2> /dev/null | sed -e "s,$BUILD_DIR,," | \
+find -L $BUILD_DIR/etc -type f 2> /dev/null | sed -e "s,$BUILD_DIR,," \
         > $BUILD_DIR/DEBIAN/conffiles
 
 # The package will be saved here (ignores <epoch>: prefix):
